src/components/SUE/report/SueReportLocation.tsx

```diff
@@ -48,6 +57,27 @@ export const SueReportLocation = ({
   const zipCodeInputRef = useRef();
   const cityInputRef = useRef();
 
+  const queryVariables = {
+    status:
+      'TICKET_STATUS_UNPROCESSED,TICKET_STATUS_OPEN,TICKET_STATUS_IN_PROCESS,TICKET_STATUS_WAIT_REQUESTOR,TICKET_STATUS_WAIT_THIRDPARTY'
+  };
```

Comment:
doesn't we have some config for these? didn't we need this statuses in the normal map as well?

Reply:
and what about `start_date: '1900-01-01T00:00:00+01:00'` here? not needed?

Reply:
No. Unfortunately, we use the `appDesignSystem` on the main-server for this. Here the status types are defined. But adding it to the app. makes sense!
---
Good point! Yes, you're right. I should add `start_date`. Otherwise we can't access all the data.